specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/bigDataPool.json

```diff
@@ -532,6 +545,44 @@
         }
       }
     },
+    "LibraryInfo": {
+      "type": "object",
+      "description": "Library/package information of a Big Data pool powered by Apache Spark",
+      "title": "Information about a library/package created at the workspace level.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the library."
+        },
+        "path": {
+          "type": "string",
+          "description": "Storage blob path of library."
+        },
+        "containerName": {
+          "type": "string",
+          "description": "Storage blob container name."
+        },
+        "uploadedTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The last update time of the library."
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of the library."
+        },
+        "provisioningStatus": {
```

Comment:
What are the possible values for provisioningStatus?

Reply:
The below are the possible values:

None,
Incomplete,
Succeeded,
Failed


However, we're tracking this as a string instead of an ENUM. Hence not marked this as an enum in swagger.